bedrock/firefox/templates/firefox/channel/desktop.html

```diff
@@ -86,3 +100,9 @@
   </footer>
 </section>
 {% endblock %}
+
+{% block js %}
+  {% if LANG.startswith('en-') and switch('mozorg-search-developer') %}
```

Comment:
This `if` statement and the one from the search block above (line 25) don't match. Seems like they should match.